frontend/src/prompted_readings/analysis_view.js

```diff
@@ -29,15 +88,20 @@ class AnalysisView extends React.Component {
         if (this.state.analysis !== null) {
             const {  // object destructuring:
                 total_view_time,
+                frequency_feelings,
+                context_vs_read_time,
                 compute_median_view_time,
             } = this.state.analysis;
-
             return (
-                <div>
+                <div style={{padding: '20px'}}>
```

Comment:
I'm going to accept this for now, just to get you moving along, but we want to use Bootstrap (which we'll talk about in the sessions next M/Tu) instead of hand writing CSS.